examples/User Creation & Provisioning/README.md

```diff
@@ -64,8 +73,8 @@ You can also specify parameters while run the command, for example:
 ```
 PS > ./Users_Create_Provision.ps1 -EmployeeList ./Employees_1.csv `
```

Comment:
Since the RootFolderName isn't required when doing Local upload, suggest to use json and pass FolderStructureJSONPath instead of LocalUploadPath flag. So we can showcase most of the params.

As if you try run the command as suggested with Json and Folderpath, you'll get an error `Please specify either a local upload path or a folder structure JSON path, not both.`